test/input_files/v4.2/passed/passed_ploidy_001.vcf

```diff
@@ -0,0 +1,6 @@
+##fileformat=VCFv4.1
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	HG00096	HG00097
+1	61462	.	T	A	100	PASS	.	GT:PL	0|0:1,2,3	1|1:4,5,6
+2	61462	.	T	A,C	100	PASS	.	GT:PL	0|0:1,2,3,4,5,6	0|1|1:10,11,12,13,14,15,16,17,18,19
```

Comment:
Maybe test genotype 0|1|2 to ensure that the index numbers don't have any effect?